inetum-realjobs-frontend/src/app/components/resume/details/skill-list/skill-list.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {Skill} from "../../../../models/skill";
+import {SkillLevel} from "../../../../models/skillLevel.enum";
+
+@Component({
+  selector: 'app-skill-list',
+  templateUrl: './skill-list.component.html',
+  styleUrls: ['./skill-list.component.scss']
+})
+export class SkillListComponent implements OnInit {
+
+  @Input() skills: Skill[];
+  @Input() deleteButton: boolean;
+  @Output() deleteEvent = new EventEmitter<number>();
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
+  }
```

Comment:
Empty `ngOnInit` should be removed. Don't forget to remove `implements OnInit` for the class.